homeassistant/components/ambient_station/binary_sensor.py

```diff
@@ -239,6 +275,34 @@ class AmbientBinarySensorDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
         on_state=0,
     ),
+    AmbientBinarySensorDescription(
+        key=TYPE_LEAK1,
+        name="Leak detector 1",
+        device_class=BinarySensorDeviceClass.MOISTURE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        on_state=1,
+    ),
+    AmbientBinarySensorDescription(
+        key=TYPE_LEAK2,
+        name="Leak detector 2",
+        device_class=BinarySensorDeviceClass.MOISTURE,
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
Curious why the leak sensors themselves are diagnostic?

Reply:
I got it from the RainMachne integration which was the only other ref to leaks I could find but on reflection I think you're right it shouldn't be diagnostic.    I will amend the PR